tools/privacyidea-token-janitor

```diff
@@ -386,72 +402,77 @@ def find(last_auth, assigned, active, tokeninfo_key, tokeninfo_value,
                                      tokeninfo_value_after,
                                      tokeninfo_value_before)
 
-    tlist = _get_tokenlist(last_auth, assigned, active, tokeninfo_key,
-                           filter, orphaned, tokentype, serial,
-                           description)
-
-    if not action:
-        if not csv:
-            print("Token serial\tTokeninfo")
-            print("="*42)
-            for token_obj in tlist:
-                print("{0!s} ({1!s})\n\t\t{2!s}\n\t\t{3!s}".format(
-                    token_obj.token.serial,
-                    token_obj.token.tokentype,
-                    token_obj.token.description,
-                    token_obj.get_tokeninfo()))
+    next = 1
+    chunksize = int(chunksize)
+    while next:
+        print("+ Reading token in chunk from database...")
```

Comment:
Hmm, either *chunks, or *tokens, I'm not sure :-)

Reply:
done